gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/diagnosticreport/SpecimenCompoundsMapper.java

```diff
@@ -49,45 +52,50 @@ public class SpecimenCompoundsMapper {
 
     private final SpecimenBatteryMapper batteryMapper;
 
-    public List<Observation> handleSpecimenChildComponents(RCMRMT030101UKEhrExtract ehrExtract, List<Observation> observations,
-                                                           List<Observation> observationComments, List<DiagnosticReport> diagnosticReports,
-                                                           Patient patient, List<Encounter> encounters, String practiseCode) {
-
+    public List<Observation> handleSpecimenChildComponents(
+        RCMRMT030101UKEhrExtract ehrExtract,
+        List<Observation> observations,
+        List<Observation> observationComments,
+        List<DiagnosticReport> diagnosticReports,
+        Patient patient, List<Encounter> encounters, String practiseCode
```

Comment:
```suggestion
        Patient patient, 
        List<Encounter> encounters, 
        String practiseCode
```

If we could keep the parameter formatting consistent I think this would be good!

Reply:
thanks!